superagi/helper/s3_helper.py

```diff
@@ -35,3 +36,17 @@ def upload_file(self, file, path):
             logger.info("File uploaded to S3 successfully!")
         except:
             raise HTTPException(status_code=500, detail="AWS credentials not found. Check your configuration.")
+
+    @classmethod
+    def get_s3_client(cls):
```

Comment:
u can use this method in this class constructor as well.

Reply:
okay